multiraft/multiraft.go

```diff
@@ -551,6 +563,12 @@ func (s *state) start() {
 				select {
 				case resp := <-s.writeTask.out:
 					s.handleWriteResponse(resp, writingGroups)
+					// Try to send events
+					select {
```

Comment:
Does this need to be here? I don't think it'll be any faster than just letting the next iteration of the main loop send it.

If we do keep this, it needs to be consistent with the main loop's version (including the `log.V(8)` message), so that log line and the clearing of `s.pendingEvents` should be factored out into a separate method.